src/components/help/help.spec.js

```diff
@@ -82,11 +82,11 @@ describe('Help', () => {
     });
   });
 
-  it('calls preventDefault to prevent clicking interacting with the input', () => {
+  it('do not prevent clicking interacting with the input', () => {
```

Comment:
Very minor obviously but should probably read "it does not prevent clicking..."

Reply:
Could we not just delete this test instead?

Reply:
We could, but I think it's best to keep it to prevent regressions. If it was a new component and new functionality, I wouldn't write a negative assertion;

Reply:
Changed inline with Sam's suggestion